_pages/qa/request-leave.md

```diff
@@ -2,6 +2,11 @@
 layout: q-and-a
 permalink: /request-leave/
 title: Request leave
+tags:
+- ooo
+- pto
+- out of office
+- paid time off
```

Comment:
For better or worse, I basically just add tags for relevant terms that aren't present on the page so they come up in search :grin: Done!